examples/pico_w/picow_wifi_scan.nim

```diff
@@ -0,0 +1,52 @@
+import std/strutils
+
+import picostdlib
+import picostdlib/pico/cyw43_arch
+
+proc scanResult(env: pointer; res: ptr Cyw43EvScanResultT): cint {.cdecl.} =
+  if not res.isNil:
+    let ssid = cast[cstring](res.ssid[0].addr)
```

Comment:
Don't know how important this is, but if any ssid is the length of the array, this is technically unsafe, since there'd be no `\0` at the end.